studio/components/interfaces/App/RouteValidationWrapper.tsx

```diff
@@ -36,14 +36,17 @@ const RouteValidationWrapper: FC = ({ children }) => {
     return excemptUrls.includes(router?.pathname)
   }
 
+  const { data: organizations, isSuccess: orgsInitialized } = useOrganizationsQuery()
+  const organizationsRef = useLatest(organizations)
```

Comment:
Because we don't want the `useEffect` to run whenever the organisations change, but when it does run, we want the organisations to be up to date. So yep no stale data :)